PowerEditor/installer/nativeLang/swedish.xml

```diff
@@ -630,14 +630,15 @@ Translation note:
 					<Item id="2219" name="Fördefinierade nyckelord"/>
 					<Item id="2221" name="Användardefinierade nyckelord"/>
 					<Item id="2225" name="Språk:"/>
-					<Item id="2226" name="Aktivera global förgrundsfärg"/>
-					<Item id="2227" name="Aktivera global bakgrundsfärg"/>
-					<Item id="2228" name="Aktivera globalt teckensnitt"/>
-					<Item id="2229" name="Aktivera global teckenstorlek"/>
-					<Item id="2230" name="Aktivera global fet teckenstil"/>
-					<Item id="2231" name="Aktivera global kursiv teckenstil"/>
-					<Item id="2232" name="Aktivera global understruken teckenstil"/>
+					<Item id="2226" name="Tvinga förgrundsfärg för alla stilar"/>
+					<Item id="2227" name="Tvinga bakgrundsfärg för alla stilar"/>
+					<Item id="2228" name="Tvinga teckensnitt för alla stilar"/>
+					<Item id="2229" name="Tvinga teckenstorlek för alla stilar"/>
+					<Item id="2230" name="Tvinga fet stil för alla stilar"/>
+					<Item id="2231" name="Tvinga kursiv stil för alla stilar"/>
+					<Item id="2232" name="Tvinga understrykning för alla stilar"/>
 					<Item id="2234" name="Gå till inställningarna"/>
+					<Item id="2235" name="Vad är global åsidosättning?"/>
```

Comment:
"Global override" should not be translated.

Reply:
Done! ✔️